setup.py

```diff
@@ -61,6 +61,8 @@
         'azdev.utilities',
     ],
     install_requires=[
+        'azure-cli-core',
```

Comment:
I searched for the code that might import `azure.cli.core`. Seems only one place is using and it's not from your modification. Then should we need to import it?